modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAbstractIntegrationTest.java

```diff
@@ -130,6 +130,13 @@ public void deleteUser(OIDCUser user) throws Exception {
      */
     public void createApplication(OIDCApplication application) throws Exception {
 
+        ServiceProvider serviceProvider = new ServiceProvider();
+        createApplication(serviceProvider, application);
+    }
+
+    public ServiceProvider createApplication(ServiceProvider serviceProvider, OIDCApplication application)
+            throws Exception {
+
         log.info("Creating application " + application.getApplicationName());
```

Comment:
Though this is not added from this PR, do we really need this info log? If not it will be good to remove it.